code/game/mecha/mecha_defense.dm

```diff
@@ -258,7 +258,7 @@
 
 	else if(istype(W, /obj/item/mecha_parts/mecha_tracking))
 		if(!user.transferItemToLoc(W, src))
-			to_chat(user, "<span class='warning'>[W] is stuck to your hand, you cannot put it in [src]!</span>")
+			to_chat(user, "<span class='warning'>[W] is stuck to your hand, you cannot put it in \the [src]!</span>")
```

Comment:
this doesn't actually fix the issue
`you cannot put it in the Durand` correct
`you cannot put it in Durand` wrong
`you cannot put it in the LUNG'S MECH` wrong
`you cannot put it in LUNG'S MECH` correct